### ISSUE TITLE
false positive no-undef with export default function

### ISSUE THREAD

**Issue by KotlinIsland** [denoland/deno_lint#643]
```ts
export default function hi() {
  console.log("hi :)");
}
{
  hi();
}
```
```
(no-undef) hi is not defined
  hi();
  ^^
```
